src/pages/Application/Loaded.tsx

```diff
@@ -0,0 +1,147 @@
+            {r.ActiveInCountries.join(", ")}
+          </Row>
+          <Row label="UN SDG">{r.UN_SDG.join(", ")}</Row>
+          <Row label="Contact name">{c.FirstName + " " + c.LastName}</Row>
+          <Row label="Contact email">{c.Email}</Row>
+          {doc.DocType === "FSA" && (
+            <>
+              <Row label="Contact national ID">
+                <DocLink url={doc.ProofOfIdentity.publicUrl} />
+              </Row>
+              <Row label="Charity registration doc">
+                <DocLink url={doc.ProofOfRegistration.publicUrl} />
+              </Row>
+              <Row label="Fiscal sponsorship agreement">
+                <DocLink url={doc.SignedFiscalSponsorshipAgreement ?? ""} />
+              </Row>
+            </>
+          )}
+        </div>
+      </Container>
+      <Container title="Banking details">
+        <div className="grid sm:grid-cols-[auto_auto_1fr]">
+          <Row label="Bank name">{w?.bankName || NA}</Row>
+          <Row label="Address">{w?.address || NA}</Row>
+          <Row label="Account number">{w?.accountNumber || NA}</Row>
+          <Row label="Account holder name">{w?.accountName || NA}</Row>
+          <Row label="Bank statement document">
+            <DocLink url={r.BankStatementFile.publicUrl} />
+          </Row>
+        </div>
+      </Container>
+      <div className="flex gap-x-3 justify-self-center sm:justify-self-end">
+        <Link
+          to={appRoutes.applications}
+          className="px-4 py-1 min-w-[6rem] font-work text-sm uppercase btn-outline"
+        >
+          back
+        </Link>
+        <button
+          disabled={!!prevVerdict}
+          onClick={review("reject")}
+          type="button"
+          className="px-4 py-1 min-w-[6rem] font-work text-sm uppercase btn-red"
+        >
+          reject
+        </button>
+        <button
+          disabled={!!prevVerdict}
+          onClick={review("approve")}
+          type="button"
+          className="px-4 py-1 min-w-[6rem] font-work text-sm uppercase btn-green"
+        >
+          approve
+        </button>
+      </div>
+    </>
+  );
+}
+
+function DocLink({ url }: { url: string }) {
+  return (
+    <ExtLink href={url} className="text-blue hover:text-blue-d1">
+      <span className="break-all">{url}</span>
+      <Icon
+        type="ExternalLink"
+        className="inline relative bottom-px ml-2"
+        size={15}
+      />
+    </ExtLink>
+  );
+}
+
+type Props = PropsWithChildren<{
+  label: string;
+}>;
+function Row(props: Props) {
+  return (
+    <>
+      <div className="px-3 max-sm:pt-3 sm:p-3 flex items-center text-xs font-semibold uppercase">
+        {props.label}
+      </div>
+      <div className="max-sm:hidden w-px border-r border-prim last:border-none" />
+      <div className="px-3 max-sm:pb-3 sm:p-3 flex items-center text-sm">
+        {props.children}
+      </div>
+      <div className="h-px col-span-full border-b border-prim last:border-none" />
```

Comment:
small nit: could [Divider.tsx](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.9/src/components/Divider.tsx) be used here instead of this `div`?

Reply:
i believe [multi-cursor editing](https://tailwindcss.com/docs/reusing-styles#multi-cursor-editing) is enough for this one